include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r3.hpp

```diff
@@ -383,7 +383,8 @@ struct GridwiseGemm_k0mk1_k0nk1_mn_xdlops_v2r3
 
     __host__ static constexpr bool CalculateHasMainKBlockLoop(index_t K)
     {
-        const index_t num_loop = K / (K0PerBlock * K1);
+        const index_t KPad     = math::integer_divide_ceil(K, K0PerBlock * K1) * K0PerBlock * K1;
```

Comment:
Minor: You can just assume `const index_t num_loop = math::integer_divide_ceil(K, K0PerBlock * K1);` But ignore this if You prefer this solution.